### PR TITLE
[Japneet | Priya] Add. payment ui

### PR DESCRIPTION
**Pull request by japneetbhatia**

### CODE REVIEW

File: src/components/Payment.css [hunk 1]
```diff
@@ -4,4 +4,20 @@
     margin-bottom: 0.5em;
     padding-top: 0.5em;
     padding-bottom: 0.5em;
-  }
\ No newline at end of file
+  }
+
+.css-1x2q1ov-MuiGrid-root {
```
[the remark this answers is not in the document]
Comment (japneetbhatia) [reply src/components/Payment.css]: Updated

File: src/components/Payment.js [hunk 2]
```diff
@@ -21,17 +40,26 @@ const CashPayment = ({ onPaymentSelect, selectedValue }) => (
           inputProps={{ 'aria-label': 'Cash' }}
         />
 )}
-      label="Cash"
+      label={(
+        <div className="payment-label">
+          <div className="img-style">
```
Comment (hariharanweb) [inline src/components/Payment.js:45]: Rename this with payment-img-style ; Something like what less css would do

File: src/components/Payment.js [hunk 3]
```diff
@@ -21,17 +40,26 @@ const CashPayment = ({ onPaymentSelect, selectedValue }) => (
...
+      )}
     />
 
   </Grid>
 );
 
 const OnlinePayment = ({ onPaymentSelect, selectedValue }) => (
 
-  <Grid container className={containerStyle} display="flex" paddingLeft={4}>
+  <Grid container className="payment-options" id="item" display="flex" paddingLeft={4}>
```
Comment (hariharanweb) [inline src/components/Payment.js:59]: Id cant be same in items ?

File: src/components/Payment.js [hunk 4]
```diff
@@ -40,23 +68,49 @@ const OnlinePayment = ({ onPaymentSelect, selectedValue }) => (
           name="radio-buttons"
           inputProps={{ 'aria-label': 'UPI' }}
         />
-
 )}
-      label="UPI"
+      label={(
+
+        <div className="payment-label">
+          <div className="img-style">
```
Comment (hariharanweb) [inline src/components/Payment.js:75]: here too

File: src/components/Price.js [hunk 5]
```diff
@@ -4,13 +4,13 @@ import Typography from '@mui/material/Typography';
 const Price = ({ price }) => (
   <>
     {!!price.value && (
-    <Typography variant="subtitle2" gutterBottom>
+    <Typography variant="body1" style={{ fontSize: 'medium', fontWeight: '1000', color: 'red' }}>
```
Comment (hariharanweb) [inline src/components/Price.js:7]: Make it in css

File: src/components/Payment.js (hunk 2 above)
Comment (japneetbhatia) [reply src/components/Payment.js:45]: Updated

File: src/components/Payment.js (hunk 4 above)
Comment (japneetbhatia) [reply src/components/Payment.js:75]: Updated

File: src/components/Price.js (hunk 5 above)
Comment (japneetbhatia) [reply src/components/Price.js:7]: Updated

**Pull request closed without merging**